config/env/default.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUploadDir: './modules/users/client/img/profile/uploads/'
```

Comment:
We still need to make changes to the config object here and also include a reference to the configuration option per environment and not hard code the upload dir destination.